GooglePlayInstant/Samples/TestApp/Scripts/TestApp.cs

```diff
@@ -21,40 +21,48 @@ namespace GooglePlayInstant.Samples.TestApp
     /// </summary>
     public class TestApp : MonoBehaviour
     {
-        private const string CookieEmoticon = "\uD83C\uDF6A";
+        private const string CookiePrefix = "test-cookie";
         private string _storedCookie;
 
         /// <summary>
         /// Sets the instant app cookie to a unique string
         /// </summary>
-        public void BtnEvt_WriteCookie()
+        public void WriteCookie()
         {
             //Write a random value so WriteCookie will always change the cookie
             var guid = Random.Range(int.MinValue, int.MaxValue);
-            _storedCookie = string.Format("{0}:{1}", guid, CookieEmoticon);
-            CookieApi.SetInstantAppCookie(_storedCookie);
-            Debug.LogFormat("Wrote a cookie: {0}", _storedCookie);
+            _storedCookie = string.Format("{0}:{1}", guid, CookiePrefix);
+            if (CookieApi.SetInstantAppCookie(_storedCookie))
+            {
+                Debug.LogFormat("Successfully wrote cookie");
+            }
+            else
+            {
+                Debug.LogError("Failed to write cookie");                
+            }
         }
         
         /// <summary>
         /// Reads the cookie and verifies if it matches the one we stored
         /// </summary>
-        public void BtnEvt_ReadCookie()
+        public void ReadCookie()
         {
+            // TODO: Currently reading the cookie from the instant app. Prefer to read it from installed app.
             var readCookie = CookieApi.GetInstantAppCookie();
-            Debug.LogFormat("Read a cookie: {0}", readCookie);
+            Debug.Log("Successfully read cookie");

```

Comment:
Sorry, I didn't comment on this line and hadn't meant for you to change this one (you can print the value).

I'm trying to think through the right balance of log messages that the automation script will read (better to be constants) vs the ones that a person will want to dig through if there is a test failure.

Reply:
That makes sense. I'll update that log.